rust/sealevel/programs/hyperlane-sealevel-igp/src/instruction.rs

```diff
@@ -117,9 +117,9 @@ pub fn init_instruction(
     let ixn = Instruction::Init;
 
     // Accounts:
-    // 0. `[executable]` The system program.
-    // 1. `[signer]` The payer account.
-    // 2. `[writeable]` The program data PDA account.
+    // 0. [executable] The system program.
```

Comment:
sorry this is from me porting this over from my v2 PR (#2778). You probably wanna keep the backticks throughout this file